packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -857,27 +861,36 @@ async function processRequest(params) {
       // reportType is in ['Inventory', 'Granule Not Found']
       await createReconciliationReport(recReportParams);
     }
-    apiRecord = await reconciliationReportModel.updateStatus({ name: reportRecord.name }, 'Generated');
-    await indexReconciliationReport(esClient, { ...apiRecord, status: 'Generated' }, process.env.ES_INDEX);
+
+    const generatedRecord = {
+      ...reportPgRecord,
+      status: 'Generated',
+    };
+    [reportPgRecord] = await reconciliationReportPgModel.upsert(knex, generatedRecord);
+    reportApiRecord = translatePostgresReconciliationReportToApiReconciliationReport(reportPgRecord); // eslint-disable-line max-len
```

Comment:
so what do you suggest? like this?
```
reportApiRecord = translatePostgresReconciliationReportToApiReconciliationReport(
  reportPgRecord
);
```
looks a bit funny (to me) but should work.

Reply:
Yeah, it's just a very long function name. I personally prefer this to disabling lint. Abbreviating words in the function name is another option but I do like descriptive titles so that gets much more nitpicky.

Reply:
Really? I wonder if your editor settings are more aggressive than the project linter settings? This for me passes linting:

```
log.error(
      `Error caught in createReconciliationReport creating ${reportType} report ${reportRecordName}. ${error}`
    );
```